stripper/ze_the_curse_of_blackwater_v6b.cfg

```diff
@@ -3,6 +3,19 @@
 ;- Nerfed Stage 2 and 3 boss hp a bit more
 ;- Fixed bug where zombies could swim on top of water on stage 3
 
+;Increase mp_freezetime to be equal to the time locked in the spawn's black box (so zombies don't spawn right as you tp out of it)
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "Cmd,Command,mp_freezetime 16,0,-1"
+	}
+}
```

Comment:
The map sets zr_infect_spawntime only 5 seconds longer than ours (15 vs 20 seconds), so adding 11 extra seconds to the freezetime seems excessive here.

Personally I would just set that 20 second infection timer in a map ZR config, but I really don't care either way.